.circleci/config.yml

```diff
@@ -7,17 +7,17 @@ orbs:
 parameters:
   cache_version:
     type: string
-    default: v5
+    default: v8
   nodejs_base_image:
     type: string
-    default: 'cimg/node:18.17.1'
+    default: 'cimg/node:20.17.0'
 
 executors:
   nodejs:
     docker:
       - image: << pipeline.parameters.nodejs_base_image >>
   docker_builder:
-    resource_class: medium
+    resource_class: large
```

Comment:
We should not do that for every container in the workflow. Most jobs should run on small ressource_class, in some rare cases, you need a larger machine ! (so if you need more resource for a job, apply a custom resource class to it)